Src/CSharpier.Tests/TestFiles/ParenthesizedLambdaExpression/ParenthesizedLambdaExpressions.cst

```diff
@@ -13,13 +13,10 @@ public class ClassName
             ) => someLongNameThatWillDoStuff
         );
 
-        var task = Task.Factory.StartNew(
-            async () =>
-            {
+        var task = Task.Factory.StartNew(async () => {
```

Comment:
Oaf. Are you sure about this? It's the first time we've opened a curly brace before the line break...

Reply:
I opened this to address it https://github.com/belav/csharpier/issues/238
But it turned out to be a two minute fix not counting the indentation.